caputilities/enigma_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
```

Comment:
change the header to hpe and update the year refer the odim code for the same

Reply:
added